src/StandaloneApp.vue

```diff
@@ -123,7 +123,8 @@ function configureAxios() {
         }
       } else {
         // show error notification
-        notificationsStore.createNotificationFromAxiosError(error)
+        if (!(error instanceof CanceledError))
+          notificationsStore.createNotificationFromAxiosError(error)
```

Comment:
Does prettier allows this? When possible, add brackets please :pray:

Reply:
Indeed it's been allowed by eslint, not good, you can leave it as it is, gonna open a PR afterwards

Reply:
whoops I've already added the brackets, don't think it's going to cause an issue though :P